edvart/report_sections/table_of_contents.py

```diff
@@ -89,8 +89,7 @@ def add_cells(self, sections: List[Section], cells: List[Dict[str, Any]]) -> Non
         # Add links to all main sections (not including subsections) besides the first (table of
```

Comment:
The part of this comment *... besides the first ...* is now redundant. As I understand it, it explained the `if isinstance..` line which is now removed.

Reply:
deleted